src/main/perl/Lock.pm

```diff
@@ -1,11 +1,5 @@
-# ${license-info}
-# ${developer-info}
-# ${author-info}
-# ${build-info}
+#${PMpre} CAF::Lock${PMpost}
```

Comment:
hmm, this is a valid notation here?
I have never seen this markup before.
Is there some pre-processing tool that understands this?